### PR TITLE
adds hardware horn implementation

### PR DESCRIPTION
**Pull request by max-grzanna**
This PR introduces a microcontroller-based implementation (actuator-provider) using the ESP32 platform and zenoh-pico.
It serves as an alternative to the software horn, providing a similar functionality for showcasing the horn’s state.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -62,6 +62,7 @@ The Zenoh router routes message between the [Kuksa-Zenoh-Provider](#kuksa-zenoh-
 git submodule update --init
 ```
 
+As an alternative you can pull the service-to-signal repository directly by executing:
```
Comment (sophokles73) [inline README.md:65]: this looks like a duplicate ...

File: README.md [hunk 2]
```diff
@@ -90,16 +91,26 @@ As a result the `horn-service-kuksa` becomes available on port 15000 on the host
 
 ```bash
 cargo run
+cargo run
```
Comment (sophokles73) [inline README.md:94]: duplicate

File: README.md [hunk 3]
```diff
@@ -90,16 +91,26 @@ As a result the `horn-service-kuksa` becomes available on port 15000 on the host
...
 ```
 
 For more details read the documentation in the [horn client Readme](./components/horn-client/README.md).
 
 > This requires that you installed the [Rust toolchain](https://rustup.rs) on your computer. As an alternative you can umcomment the section for the `horn-client` in the [service-to-signal-compose.yaml](./service-to-signal-compose.yaml) and re-deploy the modified Docker Compose setup.
 
+4. Check logs for Horn
```
Comment (sophokles73) [inline README.md:101]: duplicate

**Review by sophokles73 — CHANGES_REQUESTED** [review]

**Comment by max-grzanna** [comment]
I must have overlooked that when rebasing. Thanks for the hint.

**Review by sophokles73 — APPROVED** [review]
LGTM

**Pull request merged**